app-backend/datamodel/src/main/scala/AnnotationProjectStatus.scala

```diff
@@ -0,0 +1,65 @@
+package com.rasterfoundry.datamodel
+
+import cats.syntax.either._
+import io.circe._
+
+import java.security.InvalidParameterException
+
+sealed abstract class AnnotationProjectStatus(val repr: String) {
+  override def toString = repr
+
+  /** Order possible statuses to support annotation project workflow
+    *
+    * These values are used to filter to annotation projects with statuses
+    * no greater than a specified value, so for example, projects that
+    * have either failed to be ingested or haven't been attempted but
+    * that aren't queued or currently ingesting.
+    */
+  def toInt: Int = repr.toUpperCase match {
```

Comment:
This was an idea from when we were using airflow for job processing and I wanted a way to filter based on "jobs stuck below stage x" -- it's not currently used for anything on `IngestStatus` and I don't think it should proliferate